src/core/dbus-unit.c

```diff
@@ -2437,10 +2437,6 @@ int bus_unit_set_properties(
                 if (r < 0)
                         return r;
 
-                if (!UNIT_VTABLE(u)->bus_set_property)
```

Comment:
Yeah, it seems necessary to conditionalize the call `bus_set_property()` below.

Reply:
Duh, that was obvious.